### PR TITLE
Code restructure of router.ex

### PR DESCRIPTION
**Pull request by manuelgeek**
## What is the Purpose?
Trying to format the code, The purpose for this is to have like a controller in a different file so that the `router.ex` don't become huge

## What was the approach?
Created a 'at_sample_app_web.ex` where i have my 'router' and 'controller` functions defined
and created a 'controllers' directory

## Are there any concerns to addressed further before or after merging this PR?
The routes in 'router.ex` works fine, but i feel there's a better way of handling that, like phoenix does. also with the current implementation, dynamic routing would be very manual.
Kindly advise on this.
And any other relevant changes and/or if the refactoring is necessary / any other refactoring alternatives

### CODE REVIEW

File: lib/at_sample_app/router.ex [hunk 1]
```diff
@@ -1,29 +1,8 @@
...
-
-  plug(Plug.Parsers,
-    parsers: [:json],
-    pass: ["text/*"],
-    json_decoder: Jason
-  )
-
-  plug(:match)
-  plug(:dispatch)
+  use AtSampleAppWeb, :router
```
Comment (kamalogudah) [inline lib/at_sample_app/router.ex:2]: 😎

File: lib/at_sample_app/controllers/contact_controller.ex [hunk 2]
```diff
@@ -0,0 +1,8 @@
+defmodule AtSampleApp.ContactController do
+  @moduledoc false
+  use AtSampleAppWeb, :controller
```
Comment (kamalogudah) [inline lib/at_sample_app/controllers/contact_controller.ex:3]: 😎

**Review by kamalogudah — APPROVED** [review]
Looking good @manuelgeek , I see you are doing Phoenix like refactor.

**Comment by manuelgeek** [comment]
> Looking good @manuelgeek , I see you are doing Phoenix like refactor.

@kamalogudah  yeah, and thanks. 
I wanted to have a controller like flow, I'll proceed startd first samlple for `ussd` usecase

**Comment by TraceyOnim** [comment]
Hey @manuelgeek you can look through this docs [plugs](https://hexdocs.pm/phoenix/plug.html) on how  you can work with controller plugs.

**Comment by manuelgeek** [comment]
@TraceyOnim thatnks for the suggestions, but ...
Phoenix controller already has a lot of abstraction for you to see that work, what i want to achieve is a refactor to almost the same, which i already did, 
Maybe if you has an alternative or don't agree to it I'll revert.
In case there's none, give me the go ahead to proceed and merge this PR

**Pull request merged**